modules/ycc_vm.py

```diff
@@ -463,6 +470,19 @@ def _is_same(self, instance: dict, spec: dict):
             ]
         )
 
+        internal_ip_diff = []
+        spec_internal_ip = spec["assign_internal_ip"] if spec.get("assign_internal_ip") is not None else {}
+        spec_internal_ip_set = set(spec_internal_ip)
+        for interface in instance["network_interfaces"]:
```

Comment:
i think that we should check separate situation when number of interfaces are bigger than 1.
```python
if len(interfaces) > 1:
 # append to error
elif assign_internal_ip is None:
  pass
else:
  # check that ip is same.